kinder/pkg/data/kindnetmanifests.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package data
+
+// KindnetCNI0_5_4 is the manifest for kindnet 0.5.4
+const KindnetCNI0_5_4 = `
```

Comment:
I know this could be a stretch for this PR, but what about using the new go v1.16 embed package instead of embedding yaml strings in our codebase?

Reply:
Sounds good. It's cool!